sample/Assets/PlayServicesResolver/Editor/Resolver.cs

```diff
@@ -0,0 +1,125 @@
+﻿// <copyright file="Resolver.cs" company="Google Inc.">
+// Copyright (C) 2015 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//    limitations under the License.
+// </copyright>
+
+namespace Google.JarResolver
+{
+	using System;
+	using System.Collections.Generic;
+	using UnityEditor;
+
+	/// <summary>
+    /// Resolver provides an interface to hide the reflection calls required to
```

Comment:
nit: bit off odd indenting going on here, any chance you could fix.  Same below.